src/errors/Messages.js

```diff
@@ -112,6 +112,8 @@ const Messages = {
 
   MEMBER_FETCH_NONCE_LENGTH: 'Nonce length must not exceed 32 characters.',
 
+  GLOBAL_COMMAND: 'This method cannot be used on global application commands.',
```

Comment:
Personally, this should explain in some way that you should use a guild's command manager instead for fetching the permissions

Reply:
Do you have a suggestion for that?

Reply:
"Permissions may only be fetched or modified by calling a guild's application command manager's functions" or similar?